assets/src/dashboard/components/popoverMenu/menu.js

```diff
@@ -124,6 +126,14 @@ const Menu = ({ isOpen, currentValueIndex = 0, items, onSelect }) => {
             }
             break;
 
+          case KEYS.ESC:
+            event.preventDefault();
+            if (onSelect) {
+              // Close menu
+              onSelect(-1);
+            }
+            break;
```

Comment:
@carlos-kelly Is this a valid way to add "hit ESC" to close the popoverMenu?

Reply:
It works but I'd like to use a defined object

In `assets/src/dashboard/app/views/myStories/content/storiesView/index.js` 

We handle the `sender` as a string from `STORY_CONTEXT_MENU_ACTIONS`. Perhaps we can add a `CLOSE` option then pass that. Then we could just add an empty case with a break;

Reply:
`-1` works here because it doesn't match any of the strings.

Reply:
Yes!  I totally agree.  I added `CLOSE` to the `STORY_CONTEXT_MENU_ACTIONS` object and now that `onSelect` passes in `{ value: STORY_CONTEXT_MENU_ACTIONS.CLOSE }`.  It hits the empty case like before, but now we can specifically catch it if we wanted too by name.